### ISSUE TITLE
build breaks after FindGraphQLTags error

### ISSUE THREAD

**Issue by modosc** [relay-tools/relay-compiler-webpack-plugin#30]
i've found that certain errors seem terminal and no matter what i do i can't get webpack to rebuild certain relay files. 
for example, here's my initial error:
```
ERROR in ./app/assets/javascripts/components/Application.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Syntax Error: Expected $, found )

GraphQL request (2:30)
1:
2:   subscription JobWasCreated() {
                                ^
3:     jobWasCreated() {
```
fixing the above error causes a rebuild which then (correctly) catches a new error:
```
ERROR in ./app/assets/javascripts/components/Application.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Syntax Error: Expected Name, found )

GraphQL request (3:19)
2:   subscription JobWasCreated {
3:     jobWasCreated() {
                     ^
4:       job {
```
fixing that error causes a rebuild which gives me a `FindGraphQLTags` error:
```
ERROR in ./app/assets/javascripts/components/Application.js
Module not found: Error: Parse error: Error: FindGraphQLTags: Operation names in graphql tags must be prefixed with the module name and end in "Mutation", "Query", or "Subscription". Got `JobWasCreated` in module `Application`. in "components/Application.js"
 @ ./app/assets/javascripts/components/Application.js 5:123-171 5:399-447
 @ ./app/assets/javascripts/routes/Application.js
 @ multi @babel/polyfill url-polyfill raf/polyfill webpack-hot-middleware/client ./app/assets/javascripts/routes/Application.js
```
but once i resolve this error i end up here:
```
ERROR in ./app/assets/javascripts/components/Application.js
Module not found: Error: Can't resolve './__generated__/ApplicationJobWasCreatedSubscription.graphql' in '/Users/jon/git/swoop/app/assets/javascripts/components'
 @ ./app/assets/javascripts/components/Application.js 5:123-194 5:445-516
 @ ./app/assets/javascripts/routes/Application.js
 @ multi @babel/polyfill url-polyfill raf/polyfill webpack-hot-middleware/client ./app/assets/javascripts/routes/Application.js
ℹ ｢wdm｣: Failed to compile.
```

and no matter what i do (changing file again, changing name to something different then back) i cannot get this file built.  for comparison if i call relay from the cli it does build the file:
```
$ npx relay-compiler --src ./app/assets/javascripts --schema ./app/assets/javascripts/graphql/schema.graphql --extensions js jsx
HINT: pass --watch to keep watching for changes.

Writing js
Created:
 - ApplicationJobWasCreatedSubscription.graphql.js
Unchanged: 4 files
✨  Done in 1.23s.
```

i can't share my repo but i hope this is enough to trace down - if not i'd be happy to create a test repo if you can point me to a simplish starting point repo? thanks!

**Comment by danielholmes** [comment]
Thanks for the report @modosc . I've got a project using this starting in a few weeks and I can look into it then. There is a simplish starting repo available here if you do want to mess around with it in the meantime: https://github.com/danielholmes/relay-compiler-webpack-plugin-example

**Comment by danielholmes** [comment]
@modosc I think this is fixed now with the latest release [v1.0.0](https://github.com/danielholmes/relay-compiler-webpack-plugin/releases/tag/v1.0.0), but I don't have a project using subscriptions to test your exact case. (Note that this version is backwards incompatible with relay-compiler 1.6 and below because of the changes introduced in relay-compiler 1.7)
